hugolib/pageSort.go

```diff
@@ -275,3 +276,20 @@ func (p Pages) Reverse() Pages {
 
 	return pages
 }
+
+func (p Pages) ByParam(paramsKeyStr string) Pages {
+	key := "pageSort.ByParam." + paramsKeyStr
+
+	paramsKeyComparator := func(p1, p2 *Page) bool {
+		v1, _ := p1.Param(paramsKeyStr)
+		v2, _ := p2.Param(paramsKeyStr)
+		s1 := fmt.Sprintf("%v", v1)
```

Comment:
Use `cast.ToString` to convert to string. Don't think it will make much practical difference in this case, but then we at least have one way of doing it.